Gemfile

```diff
@@ -36,6 +36,7 @@ gem 'jbuilder', '~> 1.2'
 gem 'devise', '>= 3.2.4'
 gem 'bootstrap-sass', '~> 3.3.0'
 gem 'autoprefixer-rails'
+gem 'rubocop', require: false
```

Comment:
Smart. Moved it to the dev group block.

Thanks!

On Wed, Apr 22, 2015 at 11:56 AM, Jeremy Carey <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/Team-Fischer/chess/pull/14#discussion_r28892436>:
>
> > @@ -36,6 +36,7 @@ gem 'jbuilder', '~> 1.2'
> >  gem 'devise', '>= 3.2.4'
> >  gem 'bootstrap-sass', '~> 3.3.0'
> >  gem 'autoprefixer-rails'
> > +gem 'rubocop', require: false
>
> maybe this should be in the development group block where the quiet_assets
> gem is since i don't think we need it in test or production
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Team-Fischer/chess/pull/14/files#r28892436>.
>



-- 
Rachel Geistfeld
(952).649.0793
rachel.geistfeld@gmail.com
Rachel Likes to Read!
<http://www.youtube.com/user/RachelLikesToRead?feature=watch>